openmetadata-ui/src/main/resources/ui/src/utils/ServiceUtils.tsx

```diff
@@ -218,9 +217,6 @@ export const serviceTypeLogo = (type: string) => {
     case MessagingServiceType.Kafka:
       return KAFKA;
 
-    case MessagingServiceType.Pulsar:
-      return PULSAR;
```

Comment:
Removing pulsar from the list, since we are cleaning the messaging service connection. Pulsar is currently not implemented